inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp

```diff
@@ -152,6 +152,11 @@ void MKLDNNConcatNode::initSupportedPrimitiveDescriptors() {
         }
     }
 
+    for (size_t i = 0; i < getParentEdges().size(); i++) {
```

Comment:
Please add the comment why do we need this condition

Reply:
Done